serde-generate/Cargo.toml

```diff
@@ -28,7 +28,7 @@ serde-reflection = { path = "../serde-reflection", version = "0.3.0" }
 tempfile = "3.1"
 bincode = "1.2"
 hex = "0.4.2"
-
+libra-canonical-serialization = { git = "https://github.com/libra/libra.git", branch = "master" }
```

Comment:
don't we need testnet branch? otherwise one breaking change (LCS related)  will cause trouble for us.

Reply:
Sure, I can change that.

Reply:
done